cc3d/core/GraphicsUtils/CC3DPyGraphicsFrame.py

```diff
@@ -348,6 +348,9 @@ def __init__(self,
     def start(self, auto_shutdown: bool = True):
         """Start the event loop"""
 
+        self.draw()
```

Comment:
does this work fine in the PLayer as well?

Reply:
No effect in player. The player implementation doesn't derive from this class.

Reply:
perfect!